### PR TITLE
chore(b*): backfill `.repo-metadata.json` files

### PR DESCRIPTION
**Pull request by coryan**
Part of the work for #11519

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11595)

### CODE REVIEW

File: google/cloud/batch/v1/.repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "batch.googleapis.com",
+    "api_shortname": "batch",
+    "client_documentation": "https://cloud.google.com/cpp/reference/batch/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "",
```
Comment (dbolduc) [inline google/cloud/batch/v1/.repo-metadata.json:6]: remove?

**Review by dbolduc — APPROVED** [review]
are there specific fields we should be looking at in the review? e.g. which ones are done manually?

File: google/cloud/batch/v1/.repo-metadata.json (hunk 1 above)
Comment (coryan) [reply google/cloud/batch/v1/.repo-metadata.json]: I was able to find it by browsing the components (the documentation has nothing about support, sigh)

**Comment by coryan** [comment]
> are there specific fields we should be looking at in the review? e.g. which ones are done manually?

The issue tracker was all manual.  The "name_pretty" was based on our READMEs and may be wrong.  Ditto for the "product_documentation" field.

**Pull request merged**